DESCRIPTION

```diff
@@ -1,14 +1,26 @@
+Type: Package
 Package: LAMP
-Title: R Package Client for LAMP Platform
-Version: 1.0.0
-Authors@R: person("Division of Digital Psychiatry at Beth Israel Deaconess Medical Center.", email = "team@digitalpsych.org", role = c("aut", "cre"))
-Description: The LAMP Platform API.
-URL: https:////
-BugReports: https://///issues
+Title: API Client for LAMP Platform
+Version: 1.0.1
+Authors@R: c(person("Phil",
+                    "Henson",
+                    role = c("aut", "cre"),
+                    email = "phenson@bidmc.harvard.edu"),
+             person("Aditya",
+                    "Vaidyam",
+                    role = c("aut")),
+             person("Division of Digital Psychiatry at Beth Israel Deaconess Medical Center", 
+                    role = c("cph")))
+Description: A convenient interface for connecting with the LAMP Platform API. 
+             The LAMP Platform is a measurement-based care platform for clinicans and 
+             researchers <https://www.digitalpsych.org/lamp.html>. Use of the platform
+             has been previously published on <https://doi.org/10.1007/s41347-019-00095-w>.
+URL: https://github.com/BIDMCDigitalPsychiatry/LAMP-r
+BugReports: https://github.com/BIDMCDigitalPsychiatry/LAMP-r/issues
 Depends: R (>= 3.3)
 Encoding: UTF-8
-License: Unlicense
+License: MIT + file LICENSE
```

Comment:
We're actually "BSD 3-Clause Revised" licensed, but I'm not sure how to specify that here?

Reply:
Hmm let me check this out

Reply:
It's actually pretty easy, just have to replace with "BSD_3_clause + file LICENSE"